lib/tests/streamlit/script_interactions_test.py

```diff
@@ -64,7 +64,7 @@ def test_cached_widget_replay_rerun(self):
             """
             import streamlit as st
 
-            @st.experimental_memo(experimental_allow_widgets=True)
+            @st.cache_data(experimental_allow_widgets=True, show_spinner=False)
```

Comment:
Why are these tests changed to use `show_spinner=False`?

Reply:
The testing utilities are not able to handle containers, which seems to be a known problem:

https://github.com/streamlit/streamlit/blob/3407ec723cd6c12c085907bf743c1a27e96e3ac2/lib/streamlit/testing/element_tree.py#L991

That's why I had to deactivate the `show_spinner` in those examples, otherwise there would be a container in the element tree -> causing an exception with this testing utilities.